### PR TITLE
Add port option

### PR DESCRIPTION
**Pull request by freepenguins**
## Pull Request Checklist

**Is this in reference to an existing issue?** No

#### General

- [X] Update Changelog following the conventions laid [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [X] Tests

```
 /opt/sensu/embedded/bin/metrics-snmp-if.rb --port 160 -h my.host.name -C public -v
{"ifIndex"=>#<SNMP::VarBind:0x000000007b68a8 @name=[1.3.6.1.2.1.2.2.1.1.1], @value=#....
```

#### Purpose

Saw my last PR was incomplete and possibly could add to confusion with --prefix|-p option, so I updated each of the plugins with --port|-P and added it to the change log.

Sorry for putting this in two separate PR's.

Thanks for the quick turn-around on the last one as well!

### CODE REVIEW

File: bin/check-snmp.rb [hunk 1]
```diff
@@ -31,7 +31,8 @@ class CheckSNMP < Sensu::Plugin::Check::CLI
          default: '127.0.0.1'
 
   option :port,
-         short: '-p port',
+         short: '-P port',
```
Comment (majormoses) [inline bin/check-snmp.rb:34]: I saw your note on the changelog but I see no prefix option. Can you point me to it?

**Review by majormoses — COMMENTED** [review]
Overall looks ok other than possibly making a breaking change without needing to?

**Comment by freepenguins** [comment]
Yeah, it does, but I figured consistency would be preferred.

Please let me know if you would like for me to resubmit with it a
non-breaking change.

**Comment by majormoses** [comment]
@eheydrick thoughts? I am ok with the breaking change here. Assuming we are ok with it it should be a Breaking Change rather than a fix. https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#the-breaking-change

File: bin/check-snmp.rb (hunk 1 above)
Comment (eheydrick) [reply bin/check-snmp.rb:34]: `metrics-snmp` and `metrics-snmp-bulk` have `-p | --prefix` flags.

**Comment by eheydrick** [comment]
I think consistency is good here and I'm fine with a major release. Suspect most people won't even notice as running snmpd on a different port is pretty non-standard.

**Comment by majormoses** [comment]
K, I will merge and fixup changelog before release.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-snmp/versions/2.0.0

**Pull request merged**